app/src/main/java/de/danoeh/antennapod/ui/screen/feed/preferences/FeedSettingsFragment.java

```diff
@@ -450,7 +450,7 @@ private void setupKeepUpdatedPreference() {
 
             pref.setChecked(feedPreferences.getKeepUpdated());
             pref.setOnPreferenceChangeListener((preference, newValue) -> {
-                boolean checked = newValue == Boolean.TRUE;
+                boolean checked = newValue.equals(Boolean.TRUE);
```

Comment:
I think doing the comparison the other way around (`Boolean.TRUE.equals(...)`) would be a bit more safe because it ensures that `newValue` being `null` is not a problem. Same for the other comparisons